official/vision/data/create_coco_tf_record.py

```diff
@@ -352,7 +353,11 @@ def create_tf_example(image,
               encoded_panoptic_masks['instance_mask'])
             })
 
-  example = tf.train.Example(features=tf.train.Features(feature=feature_dict))
+  if feature_dict_len == len(feature_dict):
```

Comment:
@luotigerlsx

written this condition to avoid writing a image without any of the object annotations/caption_annotations/panoptic_annotation